flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/TaskIOMetricGroup.java

```diff
@@ -182,6 +186,10 @@ public TimerGauge getHardBackPressuredTimePerSecond() {
         return hardBackPressuredTimePerSecond;
     }
 
+    public TimerGauge getChangelogBusyTimeMsPerSecond() {
+        return changelogBusyTimeMsPerSecond;
+    }
+
```

Comment:
You have to include this value at least in the `TaskIOMetricGroup#getBusyTimePerSecond` and `TaskIOMetricGroup#getAccumulatedBusyTime`. + add a test coverage for that. Otherwise `busy + idle + backpressured` won't add up to `1000` anymore and `changelogBusyTimeMsPerSecond` won't be included in the WebUI.

Reply:
Currently, `busy time = total time - idle time - backpressure time`, I think the `changelog busy time` will be automatically calculated into the `total busy time`, I added `testConsistencyOfTime()` to verify this.

Reply:
Ahh, good point, you are right.